tests/test_chrono.py

```diff
@@ -0,0 +1,114 @@
+
+
+def test_chrono_system_clock():
+    from pybind11_tests import test_chrono1
+    import datetime
+
+    # Get the time from both c++ and datetime
+    date1 = test_chrono1()
+    date2 = datetime.datetime.today()
+
+    # The returned value should be a datetime
+    assert isinstance(date1, datetime.datetime)
+
+    # The numbers should vary by a very small amount (time it took to execute)
+    diff = abs(date1 - date2)
+
+    # There should never be a days/seconds difference
+    assert diff.days == 0
+    assert diff.seconds == 0
+
+    # 50 milliseconds is a very long time to execute this
+    assert diff.microseconds < 50000
```

Comment:
Let's not add race conditions to the CI build bots. They are sometimes very slow when running the test suite.